website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Adding fonts
+description: An intro to adding fonts to your project.
+slug: adding-fonts
+authors: iobrado
+date: 2022-04-12
+tags: [eightshift, boilerplate, fonts]
+hide_table_of_contents: false
+---
+
+Each project is different in its way. Logo, various fonts, etc. are what define the visual identity of your website. In this post, we'll cover adding fonts to a project.
+<!--truncate-->
+
+### Importing fonts into your project
+[Our documentation](/docs/basics/fonts) covers the required steps to add a font to your project, but here we'll cover the process in a bit more detail. To even start, we need a font (or two). For this example, I'll use **_SourceSansPro_** and **_NotoSerif_** downloaded from [Google Fonts](https://fonts.google.com/). These are in `.ttf` format, so convert them to `.woff` and `.woff2` however you can. We recommend the following tools:
+- https://convertio.co/ttf-woff/
+- https://cloudconvert.com/ttf-to-woff
+- https://everythingfonts.com/ttf-to-woff
+
+Fonts should go inside the **_/assets/fonts_** folder inside your theme. Paste the fonts you want to use there. You will also notice that this folder contains an **_index.js_** file. This file is used to import fonts into your project. Here's an example of how I imported my fonts:
+```js
+// SourceSansPro WOFF
+import './SourceSansPro-Bold.woff';
+import './SourceSansPro-BoldItalic.woff';
+import './SourceSansPro-Italic.woff';
+import './SourceSansPro-Regular.woff';
+import './SourceSansPro-Light.woff';
+import './SourceSansPro-LightItalic.woff';
+
+// SourceSansPro WOFF2
+import './SourceSansPro-Bold.woff2';
+import './SourceSansPro-BoldItalic.woff2';
+import './SourceSansPro-Italic.woff2';
+import './SourceSansPro-Regular.woff2';
+import './SourceSansPro-Light.woff2';
+import './SourceSansPro-LightItalic.woff2';
+
+// NotoSerif WOFF
+import './NotoSerif-Bold.woff';
+import './NotoSerif-BoldItalic.woff';
+import './NotoSerif-Italic.woff';
+import './NotoSerif-Regular.woff';
+
+// NotoSerif WOFF2
+import './NotoSerif-Bold.woff2';
+import './NotoSerif-BoldItalic.woff2';
+import './NotoSerif-Italic.woff2';
+import './NotoSerif-Regular.woff2';
+```
+
+To add these fonts as your base font and secondary font, go to the global manifest located in **_/src/Blocks_** and add the following inside `globalVariables`:
+```json
+"globalVariables": {
+	//...
+	"baseFont": "SourceSansPro",
+	"secondaryFont": "NotoSerif",
+	//...
+}
+```
+
+Next, you can create a new file called **__typography.scss_** inside your **_/assets/styles/parts/utils/_** folder and add the following:
```

Comment:
Maybe explain why this is used and what these mixins are.

Reply:
Or just link to the sassdoc chapter for `font-face` mixin.